lang/java/tools/src/test/java/org/apache/avro/tool/TestDataFileTools.java

```diff
@@ -118,6 +118,33 @@ public void testReadToJsonPretty() throws Exception {
   }
 
   @Test
+  public void testReadHeadDefaultCount() throws Exception {
+    String expectedJson = jsonData.substring(0, 20); // first 10 numbers
+    assertEquals(expectedJson,
+      run(new DataFileReadTool(), "--head", sampleFile.getPath()));
+  }
+
+  @Test
+  public void testReadHeadEquals3Count() throws Exception {
+    String expectedJson = jsonData.substring(0, 6); // first 3 numbers
+    assertEquals(expectedJson,
+      run(new DataFileReadTool(), "--head=3", sampleFile.getPath()));
+  }
+
+  @Test
+  public void testReadHeadSpace5Count() throws Exception {
+    String expectedJson = jsonData.substring(0, 10); // first 5 numbers
+    assertEquals(expectedJson,
+      run(new DataFileReadTool(), "--head", "5", sampleFile.getPath()));
+  }
+
+  @Test
+  public void testReadHeadLongCount() throws Exception {
+    assertEquals(jsonData,
+      run(new DataFileReadTool(), "--head=3000000000", sampleFile.getPath()));
+  }
+
```

Comment:
What would people prefer happens? Should zero and negative simply do nothing? Should they throw an exception saying "bad input"?